clients/unshaded/src/main/java/tachyon/client/block/LocalBlockInStream.java

```diff
@@ -109,7 +110,7 @@ protected void bufferedRead(int len) throws IOException {
 
   @Override
   public int directRead(byte[] b, int off, int len) throws IOException {
-    ByteBuffer buf = mLocalFileChannel.map(FileChannel.MapMode.READ_ONLY, getPosition(), len);
+    MappedByteBuffer buf = mLocalFileChannel.map(FileChannel.MapMode.READ_ONLY, getPosition(), len);
```

Comment:
I am open to either way. Originally, I was thinking it is easier to track which byte buffers are mapped ones.